mediapipe/tasks/ios/build_ios_framework.sh

```diff
@@ -110,21 +98,24 @@ function build_ios_frameworks_and_libraries {
   # the order of a few MBs.
 
   # Build Task Library xcframework.
-  local FRAMEWORK_CQUERY_COMMAND="-c opt --apple_generate_dsym=false ${FULL_FRAMEWORK_TARGET}"
-  IOS_FRAMEWORK_PATH="$(build_target "${FRAMEWORK_CQUERY_COMMAND}")"
+  local FRAMEWORK_CQUERY_COMMAND="-c opt --apple_generate_dsym=false --define OPENCV=source ${FULL_FRAMEWORK_TARGET}"
```

Comment:
The previous method of using a function to build and get the path to the output was returning the wrong path in the very first run after a bazel clean. It worked fine in subsequent runs.
Modified the logic to fix this issue.